GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -420,12 +442,24 @@ gdjs.HowlerSoundManager.prototype.preloadAudio = function (
 
   for (var file in files) {
     if (files.hasOwnProperty(file)) {
-      var httpRequest = new XMLHttpRequest();
-      httpRequest.addEventListener('load', onLoad);
-      httpRequest.addEventListener('error', onLoad);
-      httpRequest.addEventListener('abort', onLoad);
-      httpRequest.open('GET', file);
-      httpRequest.send();
+      // Load as sound
+      var sound = new Howl({
+        src: [file],
+        preload: true,
+        onload: onLoad,
+        onloaderror: onLoad,
+      })
+      that._loaded.sound[file] = sound;
```

Comment:
This kind of preloading sound risky for large audio file (aka "music" in GDevelop).
We're creating two howls, which sounds like useless work - and Howl are not "free', they can use audio resources which are limited.
By not setting `html5`, we also have Howler fully download and decode the whole audio file which can be costly to do for all musics.

I think we should find a way to give this control to the user or somehow automatically do that.

Reply:
Yes this is still a draft o was testing loading at the beginning. The idea is to give the user a sort of drop-down in the resources attributes "Preloading" that can be set to "No Preloading", "Preload as sound", "Preload as music", or "Preload as music and as sound". The names are pretty self explaining I think.